packages/block-library/src/columns/edit.native.js

```diff
@@ -150,6 +156,14 @@ function ColumnsEditContainer( {
 						type="stepper"
 					/>
 				</PanelBody>
+				<PanelBody>
+					<UnsupportedFooterControl
```

Comment:
Now i'm quite sure that it should be a separate component :)) 3rd time with the same props